packages/core/test/client.test.js

```diff
@@ -60,6 +60,8 @@ describe('MosaicClient', () => {
     coord.connect(client1);
     coord.connect(client2);
 
+    await Promise.all([client1.pending, client2.pending]);
```

Comment:
This seems unrelated to other changes in this PR. Is it needed?

Reply:
It's needed because the additional asynchronous prepare call in the client initialization broke the tests.